ecs-cli/modules/cli/compose/entity/service/service.go

```diff
@@ -132,8 +145,23 @@ func (s *Service) LoadContext() error {
 		if loadBalancerName != "" {
 			s.loadBalancer.LoadBalancerName = aws.String(loadBalancerName)
 		}
+		s.loadBalancers = []*ecs.LoadBalancer{s.loadBalancer}
```

Comment:
Yeah but it not worthwhile for defining another member variable for `Service`. Instead, it might be better to

```
loadBalancers = []*ecs.LoadBalancer{
   ...
}
....
s.loadBalancers = []*ecs.LoadBalancer{loadBalancer}
```